.github/workflows/build-deploy-api-v2.yaml

```diff
@@ -18,13 +18,27 @@ jobs:
         uses: actions/checkout@v2
 
       # Login the BC Gov Openshift (DPIA TOOLS Namespace)
-      - name: Cluster Login -> TOOLS Namespace
-        uses: redhat-developer/openshift-actions@v1.1
+      # - name: Cluster Login -> TOOLS Namespace
+      #   uses: redhat-developer/openshift-actions@v1.1
+      #   with:
+      #     openshift_server_url: ${{ secrets.OPENSHIFT4_SERVER_URL }}
```

Comment:
are we keeping this or just remove them. i support just delete them if we do not need

Reply:
just testing the new implementation, and I'll continue to modify if the outcome isn't what I intended or remove if it works as intended.

Reply:
I'll also update the other actions to reflect the changes made in this file once I've confirmed it's working as required.

Reply:
@binggaofw